### ISSUE TITLE
Versioning class Endpoints

### ISSUE THREAD

**Issue by MicheleATG** [cloudendpoints/endpoints-python#70]
Hi,

I migrated from v1 to v2 and I am having this issue:
I cannot perform versioning for two different services

```
test_collection_v1 = endpoints.api(name='test', version='v1')
test_collection_v2 = endpoints.api(name='test', version='v2')

```

```
@test_collection_v1.api_class()
class TestApi(remote.Service):

    @endpoints.method(
        message_types.VoidMessage,
        message_types.VoidMessage,
        name='alive',
        path='alive',
        http_method='GET')
    def alive(self, request):
        # behavior 1
        return message_types.VoidMessage()
```

```
@test_collection_v2.api_class()
class TestApiV2(remote.Service):

    @endpoints.method(
        message_types.VoidMessage,
        message_types.VoidMessage,
        name='alive',
        path='alive',
        http_method='GET')
    def alive(self, request):
        # behavior 2 (different from 1)
        return message_types.VoidMessage()
```

`application = endpoints.api_server([TestApi, TestApiV2], restricted=False)`

Error:
<img width="1261" alt="schermata 2017-07-17 alle 16 07 00" src="https://user-images.githubusercontent.com/19574472/28273442-b3f1d734-6b0e-11e7-8fcb-7d543edc0659.png">

**Comment by inklesspen** [comment]
This is the same issue as #74 and is tracked and will be fixed there.

**Comment by inklesspen** [comment]
Not the same thing as #74. However, it should be fixed by #83, which was released in 2.2.1.
